libraries/APM_Control/examples/AP_FW_Controller_test/wscript

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+def build(bld):
+
+    if bld.env.BOARD != 'linux':
+        return
+
+    # Turn off the normal AHRS so we can define a custom one
+    bld.env.DEFINES += ['AP_AHRS_ENABLED=0']
+
+    # Turn off other things that are expecting a complete AHRS
+    bld.env.DEFINES += ['AP_FRSKY_SPORT_PASSTHROUGH_ENABLED=0', 'AP_AIRSPEED_ENABLED=0']
```

Comment:
better to have:
```
bld.env.AP_LIB_REMOVE_LIBRARY['AP_FW_Controller_test'] = ['AP_AHRS']
```

Reply:
I swapped to HAL SITL in the end and used the sim backend. Slightly less hacky.